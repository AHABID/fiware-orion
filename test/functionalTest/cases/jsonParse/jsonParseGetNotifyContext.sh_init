source ../harnessFunctions.sh

dbInit CB
brokerStart CB

