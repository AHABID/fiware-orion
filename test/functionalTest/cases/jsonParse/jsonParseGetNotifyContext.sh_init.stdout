{ "dropped" : "testharness", "ok" : 1 }
T@13:38:40  paConfigActions[727]: setting trace levels to '0-255'
T@13:38:40  paConfigActions[728]: Leaving function paConfigActions
E@13:38:40  exitFunc[976]: unlink(/tmp/orion_9999.pid): No such file or directory
T@13:38:40  semInit[61]: Initialized 'req' semaphore
T@13:38:40  semInit[62]: Initialized 'mongo' semaphore
T@13:38:40  mongoSemTake[90]: mongoConnect taking the 'mongo' semaphore for 'connecting to mongo'
T@13:38:40  mongoSemTake[92]: mongoConnect has the 'mongo' semaphore
T@13:38:40  mongoConnect[152]: mongo version server: 2.4.9 (mayor: 2, minor: 4, extra: 9)
T@13:38:40  mongoSemGive[118]: mongoConnect gives the 'mongo' semaphore for 'connecting to mongo'
F@13:38:40  ***[1030]: Connected to mongo at localhost:testharness
T@13:38:40  ensureLocationIndex[381]: ensuring 2dsphere index on location.coords (tenant )
T@13:38:40  recoverOntimeIntervalThreads[398]: query() in 'testharness.csubs' collection: '{ conditions.type: "ONTIMEINTERVAL" }'
T@13:38:40  mongoSemTake[90]: recoverOntimeIntervalThreads taking the 'mongo' semaphore for 'query in SubscribeContextCollection'
T@13:38:40  mongoSemTake[92]: recoverOntimeIntervalThreads has the 'mongo' semaphore
T@13:38:40  mongoSemGive[118]: recoverOntimeIntervalThreads gives the 'mongo' semaphore for 'query in SubscribeContextCollection'
V@13:38:40  restStart[503]: Starting HTTP daemon on IPv4 0.0.0.0 port 9999
V@13:38:40  restStart[545]: Starting HTTP daemon on IPv6 :: port 9999
T@13:38:41  httpHeaderGet[96]: HTTP Header:   User-Agent: curl/7.26.0
T@13:38:41  httpHeaderGet[96]: HTTP Header:   Host: localhost:9999
T@13:38:41  httpHeaderGet[96]: HTTP Header:   Content-Type: application/json
T@13:38:41  httpHeaderGet[96]: HTTP Header:   Accept: application/json
T@13:38:41  httpHeaderGet[96]: HTTP Header:   Content-Length: 790
T@13:38:41  connectionTreat[390]: HTTP tenant: ''
T@13:38:41  restService[197]: Treating service * /NGSI10/notifyContext
T@13:38:41  restService[229]: tenant: ''
W@13:38:41  badVerbPostOnly[48]: bad verb for url '/NGSI10/notifyContext', method 'GET'
T@13:38:41  restReply[65]: Response 1: responding with 0 bytes, Status Code 405
T@13:38:41  restReply[66]: Response payload: ''
T@13:38:41  ~RegisterContextResponse[56]: destroyed
F@13:38:41  ***[948]: In sigHandler - caught signal 15
X@13:38:41  sigHandler[954]: Received signal 15
